examples/open_deep_research/run_gaia.py

```diff
@@ -32,6 +32,7 @@
 
 from smolagents import (
     CodeAgent,
+    GoogleSearchTool,
```

Comment:
If you change the available tools, then you should run again for all previous models so their results are comparable.